WebClient/src/core/window/windowView.js

```diff
@@ -15,6 +15,10 @@ var WindowView = L.Evented.extend({
         this._maxWidth = options.maxWidth || null;
         this._maxHeight = options.maxHeight || null;
 
+        this._left = null;
```

Comment:
Naming of fields is not what i expected: _left and _currentLeft do not give a hint what is stored in them.. current/persistent might be an option?